.github/workflows/NamingSystemChecker.yml

```diff
@@ -0,0 +1,17 @@
+name: IOPS-FHIR-NamingConvention-call
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  # Allows you to run this workflow manually from the Actions tab
+  
+  push:
+  
+  workflow_dispatch:
+  
+  
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "call-workflow" which references the FHIR validator repo
+  call-workflow:
+    uses: ryma2fhir/test_scripts/.github/workflows/NamingSystemChecker.yml@main
```

Comment:
does that want moving to the ipos test script repo?

Reply:
No. There is a yml file in the test-script repo that sets runs the py file. 
This yml is needed to call the test-script yml file to run every time there is a push from this repo.